client/apps/edit/client.coffee

```diff
@@ -27,10 +28,20 @@ async = require 'async'
       EditAdmin(article: @article, channel: channel)
       $('#edit-admin')[0]
     )
-    ReactDOM.render(
-      EditContent(article: @article, channel: channel)
-      $('#edit-content')[0]
-    )
+    if sd.EDIT_2
+      if channel.get('name') is 'Artsy Editorial'
+        # TODO - replace with better conditional (or channel.isEditorial),
+        # using name allows viewing the test channels in classic for interim.
+        @article.set('layout', 'standard')
```

Comment:
Confused as to why we can't use `channel.isEditorial()`? Also this kind of backfill is very easy to do so instead of writing a conditional we should go ahead and make the change to `standard`.